src/components/quests/QuestPageFooter/QuestPageFooter.tsx

```diff
@@ -1,14 +1,46 @@
+import { IconText } from "@/components/common/IconText";
+import { TimeAgo } from "@/components/common/TimeAgo/TimeAgo";
```

Comment:
```suggestion
import { IconText, TimeAgo } from "@/components/common";
```
If we add to the barrel file